src/config.c

```diff
@@ -24,6 +24,33 @@
 #include <ctype.h>
 #include <sys/stat.h>
 
+
+const struct vpn_config invalid_cfg = {
+	.gateway_host = {'\0'},
+	.gateway_port = 0,
+	.username = {'\0'},
+	.password = {'\0'},
+	.otp = {'\0'},
+	.cookie = {'\0'},
+	.realm = {'\0'},
+	.set_routes = -1,
+	.set_dns = -1,
+	.pppd_use_peerdns = -1,
+	.use_syslog = -1,
+	.half_internet_routes = -1,
+	.pppd_log = NULL,
+	.pppd_plugin = NULL,
+	.pppd_ipparam = NULL,
+	.pppd_call = NULL,
+	.ca_file = NULL,
+	.user_cert = NULL,
+	.user_key = NULL,
+	.verify_cert = -1,
```

Comment:
Now I'm checking stuff, it looks like `verify_cert` is used but never set -- nor command line nor config, so default value of 1 is always used.